src/pageObject.js

```diff
@@ -2,14 +2,38 @@
 
 class Header {
 
+  get logo() {
+    return 'Conduit logo';
+  }
 };
 
 class PageObject {
-
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  }
+  clickOnLogo() {
+    return `Click on the ${this.header.logo}`;
+  }
 };
 
 class ArticlePage extends PageObject {
+  constructor(url, slug) {
+    super(url);
+    this.url = url + slug;
+  }
+
+  get commentButton() {
+    return '[Publish comment] button';
```

Comment:
You're returning a string as a representation of a button. This is a bit misleading, as one might expect to get an actual button object or a reference to a button element in the DOM. Consider changing the method name to reflect what it actually does, for example `get commentButtonLabel`.